front/app/component-library/components/Table/Table.test.tsx

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
 
-import userEvent from '@testing-library/user-event';
```

Comment:
Guessing there was an issue with this dependency. Do we know what it is? Just curious.

Reply:
I really don't know, it could also be that something small changed about tippy. In any case I couldn't figure out what was going wrong or why the hover wasn't working, but when I tested it manually it did work, and I wanted to get this merged as soon as possible